03/app.js

```diff
@@ -7,12 +7,44 @@ const c = randomNumber(min, max);
 
 console.log(a, b, c);
 
-
-
-
-
-
-
 function randomNumber(min, max) {
     return Math.round((Math.random() * (max - min)) + min);
-}
\ No newline at end of file
+}
+
+function getSum(a, b, c) {
+    if (a > b && a > c) {
+        return (b > c) ? (a + b) : (a + c);
+    } else if (b > c) {
+        return (a > c) ? (a + b) : (b + c);
+    } else {
+        return (a > b) ? (a + c) : (b + c);
+    }
```

Comment:
Można by było zapisać te liczby do tablicy i posortować ją.
Wtedy dwie pierwsze (lub ostatnie) liczby to te, które nasz interesują.